bin/rucio-reaper2

```diff
@@ -55,6 +56,10 @@ def get_parser():
                         help='The delay (seconds) to query replicas in BEING_DELETED state.')
     parser.add_argument('--scheme', action="store", default=None, type=str,
                         help='Force the reaper to use a particular protocol/scheme, e.g., mock')
+    parser.add_argument("--auto_exclude_threshold", action="store", default=100, type=int,
```

Comment:
As discussed, we should right away also add a config_get from the config table alternatively, so we can more easily experiment with these values.

Reply:
Which one should be preferred? CLI argument or config table?